vm-virtio/src/vhost_user/blk.rs

```diff
@@ -106,6 +106,17 @@ impl Blk {
                 .map_err(Error::VhostUserSetProtocolFeatures)?;
         }
 
+        // Get the max queues number from backend, and the queue number set
+        // should be less than this max queue number.
+        let max_queues_num = vhost_user_blk
+            .get_queue_num()
+            .map_err(Error::VhostUserGetQueueMaxNum)?;
+
+        if vu_cfg.num_queues > max_queues_num as usize {
```

Comment:
Is it not allowed to have vu_cfg.num_queues equal to max_queues_num?

Reply:
vu_cfg.num_queues can be equal to max_queues_num. thanks!